rackcity/models/rack.py

```diff
@@ -22,6 +22,7 @@ class Rack(models.Model):
     )
     rack_num = models.PositiveIntegerField()
     height = models.PositiveIntegerField(default=42)
+    is_powered = models.BooleanField(default=False)
```

Comment:
This is a little picky but I don't think `is_powered` is an accurate name since all racks have PDUs - I think something like `has_networked_pdu` or `is_networx_pro` or `is_network_controlled` would be better

Reply:
Sounds good, I'll update and push again